dotnet/src/Connectors/Connectors.AzureOpenAI.UnitTests/Services/AzureOpenAIAudioToTextServiceTests.cs

```diff
@@ -221,7 +221,6 @@ public async Task GetTextContentGranularitiesWorksCorrectlyAsync(string[]? granu
 
     public static TheoryData<string?, string?> Versions => new()
     {
-        { null, "2024-08-01-preview" },
```

Comment:
Is this version no longer supported?

Reply:
It's supported, it's just not used as default anymore on Azure .NET SDK side.